homeworks/homework-1/src/models/news-item.model.ts

```diff
@@ -0,0 +1,17 @@
+export interface NewsItemProperties {
+  id: number;
+  title: string;
+  votes: number;
+}
+
+export class NewsItemModel {
+  id: number;
+  title: string;
+  votes: number;
+
+  constructor({ id, title, votes = 0 }: NewsItemProperties) {
+    this.id = id;
+    this.title = title;
+    this.votes = votes;
+  }
+}
```

Comment:
```suggestion
  compareTo(other: NewsItemModel) : Number {
    return other.votes - this.votes
  }
}
```

Then you can later do:
```js
.sort((a, b) => a.compareTo(b));
```

Just an idea :wink: